README.md

```diff
@@ -60,14 +52,101 @@ Spinal cord MRI data:
 - T2star tra
 Brain MRI data:
 - DWI (A-P and P-A phase encoding)
-- MPM (multi-parameter mapping)
+- T1w
 
 ## naming convention
 
 sub-<site><pathology>XXX
 
 example:
-sub-zurichDCM001
+sub-montrealDCM001
+
+<ADDITIONAL NOTES IF AVAILABLE>Note: the label `bp-cspine` is used to differentiate spine images from brain.
+
+## derivatives
+
+The derivatives will be organized according to the following:
+
+https://github.com/ivadomed/ivadomed/wiki/repositories#derivatives
+
+Convention for derivatives JSON metadata:
+
+{
+  "Author": "Firstname Lastname",
+  "Date": "YYYY-MM-DD HH:MM:SS"
+}
+
+## <IF DATA ARE MISSING FOR SOME SUBJECT(S)>missing data
+
+```
+
+</details>
+
+## `dataset_description.json`
+
+The `dataset_description.json` is a JSON file describing the dataset.
+
+<details><summary>dataset_description.json template:</summary>
+
+```json
+{
+    "BIDSVersion": "BIDS 1.8.0",
+    "Name": "<dataset_name>"
+}
+```
+
+</details>
+
+## `participants.tsv`
+
+The `participants.tsv` is a TSV file and should include the following columns:
+
+| participant_id | pathology | data_id | institution_id | institution |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| sub-001 | HC | 001 | 01 | montreal |
+
+- `participant_id` - unique participant ID
+- `pathology` - pathology name; can take values listed in the [pathology column](https://docs.google.com/spreadsheets/d/1yjcA8Z0COn4OZxusIDHjStH2DpeXvscsj-aWE2X-_sg/edit?usp=sharing)
+- `data_id` -  subject ID used to locate the unprocessed data as under `duke/mri/`
+- `institution_id` - institution ID used to locate the unprocessed data as under `duke/mri/`
+- `institution` - human-friendly institution name
+- others - if available, include also demographic characteristics such as `age`, `sex`, `height`, `weight`, `researcher`, and [additional columns](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file).
```

Comment:
Several times now I've seen these extra columns cause `bids-validator` to fail. But that doesn't mean we should remove the columns! It just means that the `participants.json` file needs more entries. Could you add an example of adding one of these column descriptions to the `participants.json` section below?

Reply:
Good point! I added an explanation.

```suggestion
- others - if available, include also demographic characteristics such as `age`, `sex`, `height`, `weight`, `researcher`, and [additional columns](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file).

❗️If you use some extra column (such as `age`, `sex`, etc), you have to include an appropriate entry also to the `participants.json` file. Otherwise, `bids-validator` fails.
```